### DISCUSSION TITLE
who the f**k create this

### DISCUSSION THREAD

**Discussion by roma200802** [KadeArchive/Kade-Engine#2164] [General]
why can't I figure this engine out? I want to help my friend who wants a mod about himself but he has a weak computer. I only have one word ... REMOVE THE CREATION THROUGH THE COMMAND LINE !!!!!!!!!!!!!!!!!!!!!!!!!

**Comment by isakube** [comment]
that's how you compile a Lime project, dumbass. cry about it.

**Reply by roma200802** [reply]
...

**Reply by libracrumbs** [reply]
so true, that is the way to compile it.

**Comment by napstaa967** [comment]
> why can't I figure this engine out? I want to help my friend who wants a mod about himself but he has a weak computer. I only have one word ... REMOVE THE CREATION THROUGH THE COMMAND LINE !!!!!!!!!!!!!!!!!!!!!!!!!

step 1: open notepad
step 2: type in "lime test windows"
step 3: ctrl+s
step 4: select "all file types"
step 5: set the name to "build.bat"
step 6: save it on the root of the fnf directory
step 7: double click it
done

**Comment by roma200802** [comment]
I have not words about this...

**Comment by onyxiarivera** [comment]
dude if you're gonna be developing literally anything you need to learn how to use a command line
